app/components/HeaderNotiIcon.tsx

```diff
@@ -184,8 +184,19 @@ const HeaderNotiIcon = (props: any): any => {
               })
             }
             if (currentChildNotis.vcnotis) {
+              // 
               currentChildNotis.vcnotis.forEach((item: any) => {
+                console.log(item,"..currentChildNotis");
+                if(item.title=="vcNoti1"){
+                const vcNotisExists= getVaccinesForPeriodCount(allVaccineData, item.growth_period);
+                console.log(vcNotisExists,"..vcNotisExists..")
+                if(vcNotisExists!="" && vcNotisExists!=null && vcNotisExists!=undefined){
```

Comment:
null and undefined represent the absence of a value, while an empty value typically refers to an empty string ('') or an empty array ([]) so it is recommended to check for null or undefined first before checking for an empty value.